docs/specs/toc.yml

```diff
@@ -1605,3 +1605,113 @@ outputs:
         "a.md": [{ "source": "TOC.yml", "type": "metadata" }]
       }
     }
+---
+# TOC split metadata supporting
+repos:
+  https://ms.com/ops-file-metadata#master:
+  - files:
+      docfx.yml: |
+        fileMetadata:
+          key:
```

Comment:
remove this since this has been covered by other test

Reply:
OK. 👌